pwa/pages/about/index.tsx

```diff
@@ -0,0 +1,120 @@
+      </p>
+      <p>
+        Si nous estimons que votre paiement a violé une loi ou l’une de nos conditions d’utilisation, nous nous
+        réservons le droit d’annuler votre transaction.
+      </p>
+      <h4>Services</h4>
+      <p>Les services seront facturés en totalité à la commande du service.</p>
+      <h4>Abonnements</h4>
+      <p>
+        Tous nos abonnements récurrents seront automatiquement facturés et renouvelés jusqu’à ce que nous recevions
+        d’avis que vous souhaitez annuler l’abonnement.
+      </p>
+      <h3>Limitation de responsabilité</h3>
+      <p>
+        Sylvain COMBRAQUE ou l’un de ses employés sera tenu responsable de tout problème découlant de ce site.
+        Néanmoins, Sylvain COMBRAQUE et ses employés ne seront pas tenus responsables de tout problème découlant de
+        toute utilisation irrégulière de ce site.
+      </p>
+      <h3>Indemnité</h3>
+      <p>
+        En tant qu’utilisateur, vous indemnisez par les présentes Sylvain COMBRAQUE de toute responsabilité, de tout
+        coût, de toute cause d’action, de tout dommage ou de toute dépense découlant de votre utilisation de ce site ou
+        de votre violation de l’une des dispositions énoncées dans le présent document.
+      </p>
+      <h3>Lois applicables</h3>
+      <p>
+        Ce document est soumis aux lois applicables en France et vise à se conformer à ses règles et règlements
+        nécessaires. Cela inclut la réglementation à l’échelle de l’UE énoncée dans le RGPD.
+      </p>
+      <h3>Divisibilité</h3>
+      <p>
+        Si, à tout moment, l’une des dispositions énoncées dans le présent document est jugée incompatible ou invalide
+        en vertu des lois applicables, ces dispositions seront considérées comme nulles et seront retirées du présent
+        document. Toutes les autres dispositions ne seront pas touchées par les lois et le reste du document sera
+        toujours considéré comme valide.
+      </p>
+      <h3>Modifications</h3>
+      <p>
+        Ces conditions générales peuvent être modifiées de temps à autre afin de maintenir le respect de la loi et de
+        refléter tout changement à la façon dont nous gérons notre site et la façon dont nous nous attendons à ce que
+        les utilisateurs se comportent sur notre site. Nous recommandons à nos utilisateurs de vérifier ces conditions
+        générales de temps à autre pour s’assurer qu’ils sont informés de toute mise à jour. Au besoin, nous informerons
+        les utilisateurs par courriel des changements apportés à ces conditions ou nous afficherons un avis sur notre
+        site.
+      </p>
+      <h3>Contact</h3>
+      <p>
+        Veuillez communiquer avec nous si vous avez des questions ou des préoccupations. Nos coordonnées sont les
+        suivantes :
+      </p>
+      <p>Téléphone : 06 06 06 06 06</p>
+      <p>
+        Email :{' '}
+        <Link href="mailto:jointure.miraculeux_08@icloud.com" className="text-blue-500">
```

Comment:
Arbitrary text color. Please use the daisy colors https://daisyui.com/docs/colors/